src/Managed.hs

```diff
@@ -1,5 +1,6 @@
 module Managed where
 
+import Interfaces
 import Lookup
 import Obj
 import Types
```

Comment:
The function isExternalType here should be removed, we don't want to care about that anymore.